Source/FaceFX/Private/FaceFXCharacter.cpp

```diff
@@ -706,6 +713,120 @@ bool UFaceFXCharacter::Load(const UFaceFXActor* Dataset)
 	    }
     }
 
+	bDisabledMorphTargets = IsDisabledMorphTargets;
+	if(!bDisabledMorphTargets && !SetupMorphTargets())
+	{
+		Reset();
+		return false;
+	}
+
+	return true;
+}
+
+void UFaceFXCharacter::ProcessMorphTargets()
+{
+	checkf(MorphTargetNames.Num() == MorphTargetTrackValues.Num(), TEXT("Morph target names indices must match the track values buffer"));
+
+	const int32 MorphTargetsToProcess = MorphTargetTrackValues.Num();
+	if(MorphTargetsToProcess == 0)
+	{
+		//no morph target to process
+		return;
+	}
+
+	//read track values
+	if(!Check(ffx_read_frame_track_values(FrameState, &MorphTargetTrackValues[0], MorphTargetsToProcess)))
```

Comment:
Are these MorphTargetTrackValues sorted by increasing FaceFX track index? If not, and you run in debug mode, the FaceFX Runtime will issue a performance assert that they should be ordered.